charts/.ci/scripts/local-ct-lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I actually would prefer to have this in the `Makefile` or at least be called by the `Makefile` and otherwise I need to remember myself to call this script :)